webpack.config.babel.js

```diff
@@ -126,18 +146,37 @@ const config = {
   }
 };
 
-if (ENV === 'production') {
-  config.plugins.push(
-    new webpack.optimize.UglifyJsPlugin({
-      compressor: {
-        pure_getters: true,
-        unsafe: true,
-        unsafe_comps: true,
-        screw_ie8: true,
-        warnings: false
+
+
+const configNode = {
+  ...config,
+  name: 'node-library',
+  output: {
+    library: 'Onfido',
+    libraryTarget: 'umd',
+    path: `${__dirname}/lib`,
+    publicPath: '/',
+    target: 'web',
+    filename: 'index.js'
+  },
+  module: {
+    loaders: [
+      ...baseLoaders,
```

Comment:
I don't think the `baseLoaders` etc. are necessary are they? You could just do `...config.module.loaders` here, I think?